jac-uk/admin
# Fix Rich Text Editor

## What's included?
Removed ck editor refs from admin as they belong in the jac-kit and fixed issues with rich text editor input text.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Ensure the rich text editor works ok.

- Open a draft exercise
- Select 'Website Listing' from the side navigation panel
- Select the 'Update website listing' link from the right hand side
- Scroll down to the 'Brief Overview' and edit the text
- Scroll to the bottom of the page and press the 'Save And Continue' button
- Check that the change has been saved in the 'Brief Overview' section

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
